interchain/interchain-tests/src/bin/setup_funds.rs

```diff
@@ -95,7 +95,9 @@ pub fn test_send_funds() -> AnyResult<()> {
         },
     )?;
 
-    interchain.check_ibc(STARGAZE, send_funds_tx)?;
+    interchain
+        .check_ibc(STARGAZE, send_funds_tx)?
+        .into_result()?;
```

Comment:
Doesn't check_ibc already check result?

Reply:
Yes but it returns a `must_use` type. I brought this up to Nicolas, think cw-orch has a PR with a fix.